SPECS-EXTENDED/supermin/supermin.spec

```diff
@@ -15,12 +15,7 @@ Distribution:   Mariner
 #
 # See also:
 # https://github.com/libguestfs/supermin/commit/9bb57e1a8d0f3b57eb09f65dd574f702b67e1c2f
-
-%ifarch aarch64 %{arm} %{ix86} %{power} s390x x86_64
 %bcond_without dietlibc
-%else
-%bcond_with dietlibc
-%endif
 
 # Whether we should verify tarball signature with GPGv2.
 %global verify_tarball_signature %{nil}
```

Comment:
Do I understand this correctly that we skip the pgp signature check? Is the signature included above used then? It seems we'd never check that. In that case we should either remove signature or enable the check IMHO.

Reply:
Will fix, thanks